types/tslint.json

```diff
@@ -9,6 +9,7 @@
     "semicolon": false,
     "unified-signatures": false,
     "whitespace": false,
-    "interface-over-type-literal": false
+    "interface-over-type-literal": false,
+    "no-unnecessary-generics": false
```

Comment:
Option 3 from https://github.com/unifiedjs/unified/pull/58/files#r299705792 would allow this rule to be re-enabled.

Reply:
I'm in. This option is kind of helpful but really annoying. After adopting this, all we need to do is more careful when reviewing.